mixer/adapter/stackdriver/contextgraph/send.go

```diff
@@ -182,7 +182,7 @@ func (h *handler) call(ctx context.Context, req *contextgraphpb.AssertBatchReque
 		s, _ := status.FromError(err)
 		if d := s.Proto().Details; len(d) > 0 {
 			// Log the debug message, if present.
-			h.env.Logger().Errorf("STATUS: %s\n", d[0].Value)
+			_ = h.env.Logger().Errorf("STATUS: %s\n", d[0].Value)
```

Comment:
Seems like should be getting logged with Debugf perhaps?

Reply:
I don't know... I left it as it is since not sure what the purpose of this log line is.